framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -188,6 +190,38 @@ public int runTests() throws Exception {
 
       LOG.info("> RUNNING TESTS (ITERATION " + i + ")..");
       Collections.shuffle(tests, new Random());
+      if (cmdParam.repeatRun == true) {
+        String queryLogFilename = drillTestDataDir + "/queryLog.log";
```

Comment:
Can we store the query file inside drill-out dir? This is where we put other tmp files, so keep it consistent would be better.